src/test/kotlin/com/bardsoftware/backend/fleet/rmarkdown/DockerProcessorTest.kt

```diff
@@ -0,0 +1,22 @@
+package com.bardsoftware.backend.fleet.rmarkdown
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class DockerTest {
+    private val dockerProcessor = DockerProcessor()
+
+    @Test
+    fun testMessageSum() {
+        val messages = listOf("test", "hello", "", "0123")
+        val answers = listOf(
+                "d8e8fca2dc0f896fd7cb4cb0031ba249  -\n",
+                "b1946ac92492d2347c6235b4d2611184  -\n",
+                "68b329da9893e34099c7d8ad5cb9c940  -\n",
+                "e5870c1091c20ed693976546d23b4841  -\n")
+
+        for ((index, _) in messages.withIndex()) {
```

Comment:
If you only need index, you can use `for (index in 0 until answrs.size)`, 
but I see that you actually need `messages` values and access them with `messages[index]`. Why don't you use one which you currently forget in `_`?

But in fact, unleash the full power of Kotlin with ([`messages zip answers`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/zip.html)).forEach

Reply:
Done